convert/convert.go

```diff
@@ -210,3 +223,56 @@ func getWigChromIndex(s string, wigSlice []*wig.Wig) int {
 	log.Fatalf("Bed Chromosome, %s, not in reference genome.", s)
 	return -1
 }
+
+//PairwiseFaToVcf takes in a pairwise multiFa alignment and returns Vcf entries for segregating sites with the first entry as the reference and the second fasta entry as the alt allele.
+//This will have to be done by chromosome, as a pairwise multiFa will only have two entries, thus containing one chromosome per file.
+func PairwiseFaToVcf(f []*fasta.Fasta, chr string) []*vcf.Vcf {
+	var pastStart bool = false //bool check to see if we have an insertion at the start of an alignment.
+	var insertion bool = false
+	var deletion bool = false
+	var insertionAlnPos int
+	var deletionAlnPos int
+	answer := make([]*vcf.Vcf, 0)
+
+	for i := 0; i < len(f[0].Seq); i++ { //loop through reference alignment positions
+		if f[0].Seq[i] == dna.Gap { //reference is gap (insertion)
+			if pastStart {
+				if !insertion {
+					insertionAlnPos = i - 1 //TODO: breaks for alignments starting with an insertion(gap in Aln postion 0 of the reference)
+				}
+				insertion = true
+			}
+		} else if f[0].Seq[i] != f[1].Seq[i] {
+			pastStart = true
+			if insertion { //catches the case where an insertion, now complete, is followed directly by a snp.
+				answer = append(answer, &vcf.Vcf{Chr: chr, Pos: int64(fasta.AlnPosToRefPos(f[0], insertionAlnPos) + 1), Id: ".", Ref: dna.BaseToString(f[0].Seq[insertionAlnPos]), Alt: dna.BasesToString(f[1].Seq[insertionAlnPos:i]), Qual: 100.0, Filter: "PASS", Info: ".", Format: ".", Notes: "."})
+			}
+			if f[1].Seq[i] == dna.Gap { //alt is gap (deletion)
```

Comment:
I am getting a little thrown off by the last of symmetry between insertions and deletions.  Can you add a few comments or an overview of your thought process?  I am having a little trouble wrapping my head around the logical flow inside of this for-loop that looks at each base in the alignment.